WordPress/Tests/WP/CronIntervalUnitTest.inc

```diff
@@ -160,3 +160,45 @@ add_filter( 'cron_schedules', function ( $schedules ) {
 	];
 	return $schedules;
 } ); // Error: 8 min.
+
+// Correctly handle fully qualified WP time constants.
+class FQNConstants {
+	public function add_schedules() {
+		add_filter( 'cron_schedules', array( $this, 'add_weekly_schedule' ) );
+		add_filter( 'cron_schedules', array( $this, 'add_eight_minute_schedule' ) ); // Warning: 8 min.
```

Comment:
```suggestion
		add_filter( 'cron_schedules', array( $this, 'add_eight_minute_schedule' ) ); // Warning: 8 min, which is less than the 15 minutes default from `$this->min_interval`.
```

Doesn't have to be this exact wordy comment, but it took me a while to understand why this would be a warning, since the fix addressed this exact syntax parsing.

```suggestion
		add_filter( 'cron_schedules', array( $this, 'add_eight_minute_schedule' ) ); // Warning (CronSchedulesInterval): 8 min.
```

..would also work, for instance, since there are multiple warning violation codes for this sniff.

Reply:
The two types of warnings in the test case file are distinguished from each other by `Warning: time undetermined` vs `Warning: # min.` This is the same throughout the file, so changing that format should be done for the whole file, not for a new test only.

And yes, while the false positive I was addressing is basically what's in the first new test - `add_weekly_schedule()` -, I added the extra tests to make sure I would not be introducing _false negatives_ with the fix I was making.

Using "which is less than the 15 minutes default from `$this->min_interval`" would make things less clear as the `$min_interval` is a configurable `public` property and the test file contains tests with different settings for `min_interval`.

What about if I add ` // Ok: > 15 min.` to the first new test case ? That should help make the distinction clearer. Would that work for you ?

Reply:
No change necessary at this point - I think I was just seeing the warning message out of context of understanding there _was_ a minimum interval.

Reply:
I'd added the `// Ok: > 15 min.` in the previous update anyway, so, hopefully this will help next time someone looks at it.